example/ios/BringgDriverSdkIosExample.xcodeproj/project.pbxproj

```diff
@@ -204,6 +204,7 @@
 				"${PODS_ROOT}/BringgDriverSDK/BringgDriverSDK.framework",
 				"${BUILT_PRODUCTS_DIR}/BringgDriverSDKObjc/BringgDriverSDKObjc.framework",
 				"${BUILT_PRODUCTS_DIR}/CryptoSwift/CryptoSwift.framework",
+				"${BUILT_PRODUCTS_DIR}/DeviceKit/DeviceKit.framework",
```

Comment:
Why this framework was added?

Reply:
In the latest native sdk version (1.10.0) we extracted DeviceKit from having the code internally to a dependency.